### PR TITLE
chore(staging values): update staging with new test contract

### PR DESCRIPTION
**Pull request by jrhender**

### CODE REVIEW

File: src/environments/environment.staging.ts [hunk 1]
```diff
@@ -11,17 +11,25 @@ const env: Partial<typeof commonEnv> = {
   cacheServerUrl: 'https://identitycache.energyweb.org/v1',
   natsServerUrl: 'https://identityevents.energyweb.org/',
   kmsServerUrl: 'https://kms.energyweb.org/connect/new',
-  stakingPoolFactoryAddress: '0x630FA5B60cF5A85321cC9E8cc1e6A32f4dF73e8e',
+  stakingPoolFactoryAddress: '0xf66Cd33d68b4fF3Ab00a122e59b8B6d50B9C97eC',
```
Comment (jrhender) [inline src/environments/environment.staging.ts:14]: This is the new test contract that we deployed

**Review by dawidgil — APPROVED** [review]

File: src/environments/environment.staging.ts (hunk 1 above)
Comment (dawidgil) [reply src/environments/environment.staging.ts:14]: but wait, you updating develop?

Comment (dawidgil) [reply src/environments/environment.staging.ts:14]: do you want to test it on develop? anyway if you want to check contract locally you can copy paste configuration into environment.common.ts

Comment (dawidgil) [reply src/environments/environment.staging.ts:14]: and run ng serve

**Pull request merged**